recipe/deploy/vendors.php

```diff
@@ -1,23 +1,28 @@
 <?php
+
 namespace Deployer;
 
 set('composer_action', 'install');
 
 set('composer_options', '--verbose --prefer-dist --no-progress --no-interaction --no-dev --optimize-autoloader');
 
-// Returns Composer binary path in found. Otherwise try to install latest
-// composer version to `.dep/composer.phar`. To use specific composer version
-// download desired phar and place it at `.dep/composer.phar`.
+// If Composer is installed and version >= 2.0.0.: Returns Composer binary path in found.
+// Otherwise try to install latest composer version to `.dep/composer.phar`.
+// To use specific composer version, download desired phar and place it at `.dep/composer.phar`.
 set('bin/composer', function () {
     if (test('[ -f {{deploy_path}}/.dep/composer.phar ]')) {
-        return '{{bin/php}} {{deploy_path}}/.dep/composer.phar';
+        $composerVersion = run('{{bin/php}} {{deploy_path}}/.dep/composer.phar --version');
+        if (version_compare($composerVersion, '2.0.0', '>=')) {
```

Comment:
Not sure if it is a good solution. Let's keep the version what user installed by default.